config.sample.json

```diff
@@ -1,10 +1,15 @@
 {
-    "contract": "0x47EbaB13B806773ec2A2d16873e2dF770D130b50",
+    "contracts": [
+        "0x47EbaB13B806773ec2A2d16873e2dF770D130b50",
+        "0x1430194905301504e8830ce4B0b0df7187E84AbD",
+        "0x710B3303fB508a84F10793c1106e32bE873C24cd",
+        "0x388C818CA8B9251b393131C08a736A67ccB19297"
+    ],
     "network": "mainnet",
     "github_repo": "https://github.com/lidofinance/lido-dao/tree/master",
     "dependencies": {
-        "@aragon": "https://github.com/aragon/aragonOS/tree/f3ae59b00f73984e562df00129c925339cd069ff",
-        "openzeppelin-solidity": "https://github.com/OpenZeppelin/openzeppelin-contracts/tree/06e265b38d3e9daeaa7b33f9035c700d6bc0c6a0",
-        "solidity-bytes-utils": "https://github.com/GNSPS/solidity-bytes-utils/tree/9776282d181839fbb4b18f2cf218e316d6df871c"
+        "@aragon": "https://github.com/aragon/aragonOS/tree/v4.4.0",
```

Comment:
We use `4.4.1`
https://github.com/lidofinance/lido-dao/blob/master/package.json#L120